src/data/markdown/docs/02 javascript api/07 k6-experimental/03 websockets/10 WebSocket/10-WebSocket-send-data-.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'WebSocket.send(data)'
+excerpt: 'Send a data string through the connection.'
+---
+
+Send a data string through the connection.
+You can use `JSON.stringify` to convert a JSON or JavaScript values to a JSON string.
+
+| Parameter | Type   | Description       |
+| --------- | ------ | ----------------- |
+| data      | string | The data to send. |
```

Comment:
I'm curious, why isn't sending `ArrayBuffer` or `TypedArray` supported? It's possible in the [web spec](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/send#parameters).

Should this be a feature request in the issue tracker?

Reply:
I've checked the code and see that `ArrayBuffer` is also supported, so adding it